common/djangoapps/student/views.py

```diff
@@ -1754,6 +1755,9 @@ def create_account_with_params(request, params):
             }
         )
 
+    # Announce registration
+    REGISTER_USER.send(sender=None, user=user, profile=profile)
```

Comment:
I tried using post save, but ended up having trouble differentiating new user registration from profile updates.  I also tried having email_marketing import the signal but had problems with python handing files importing from each other.  I can play with it more.